packages/vector_graphics_compiler/lib/src/vector_instructions.dart

```diff
@@ -85,7 +92,7 @@ class DrawCommand {
   /// [VectorInstructions.vertices].
   ///
   /// Use [type] to determine which type of object this is.
-  final int objectId;
+  final num objectId;
```

Comment:
nit: I don't like num. I assume this can go away with changes mentioned above for saveLayer.

Reply:
Done